modules/photons_control/device_finder.py

```diff
@@ -496,7 +498,7 @@ class Device(dictobj.Spec):
 
     product_id = dictobj.Field(sb.integer_spec, wrapper=sb.optional_spec)
 
-    cap = dictobj.Field(sb.listof(sb.string_spec()), wrapper=sb.optional_spec)
+    cap = dictobj.Field(sb.any_spec, wrapper=sb.optional_spec)
```

Comment:
```suggestion
    cap = dictobj.Field(sb.or_spec(sb.listof(sb.string_spec()), sb.typed(Capability)), wrapper=sb.optional_spec)
```

Reply:
or can we make it so it's only a capability object and never a list?

Reply:
Not without breaking Interactor and the tests and anyone else who's relying on the output of "cap" to be what it was before.

Reply:
where is the Device object used again?

Reply:
The Device Finder reference is a list of Devices, so it's pretty important to keep it responding like it always does.